libchisel/src/dropsection.rs

```diff
@@ -29,6 +32,34 @@ impl<'a> ChiselModule<'a> for DropSection {
     }
 }
 
+impl From<std::num::ParseIntError> for ModuleError {
```

Comment:
Could make this more generic, and support all kinds of errors:
```
impl From<E: Error> for ModuleError {
    fn from(error: E) -> Self {
        ModuleError::Custom(error.description().to_string())
    }
}

Reply:
True, though I was considering here to wrap it with some text or use `NotSupported`.

Reply:
Apparently this is unstable:  https://github.com/rust-lang/rust/issues/52662

Reply:
You could have this blanket impl and then have specialized impls for specific traits